consensus/spos/bls/subroundEndRound.go

```diff
@@ -970,12 +972,16 @@ func (sr *subroundEndRound) updateMetricsForLeader() {
 }
 
 func (sr *subroundEndRound) broadcastBlockDataLeader(sender []byte) error {
+	// TODO[cleanup cns finality]: remove this method, block data was already broadcast during subroundBlock
```

Comment:
will this be done in a future PR? maybe set the comment only when the changed wil be implemented

Reply:
it is already implemented.. body and header are sent on subround block. they won't be modified after